railties/lib/rails.rb

```diff
@@ -67,7 +67,7 @@ def root
     #   Rails.env.development? # => true
     #   Rails.env.production? # => false
     def env
-      @_env ||= ActiveSupport::StringInquirer.new(ENV["RAILS_ENV"] || ENV["RACK_ENV"] || "development")
+      @_env ||= ActiveSupport::StringInquirer.new(ENV["RAILS_ENV"].presence || ENV["RACK_ENV"].presence || "development")
```

Comment:
`presence` is not present in this file. We need to require it. The test suite is passing because some of part of the test suite is requiring it and it is being used.